Pyjion/absint.cpp

```diff
@@ -2246,7 +2321,17 @@ JittedCode* AbstractInterpreter::compile_worker() {
             case BUILD_MAP_UNPACK:
                 extend_map(oparg);
                 inc_stack();
-                break;
+                break;/*
```

Comment:
I'll move it to where we currently handle the case, but leave it commented out.  It still needs to be implemented